cli/openApiSpec.test.ts

```diff
@@ -0,0 +1,25 @@
+import fs from 'fs';
+import * as ts from 'typescript';
+import { generateOpenApiSpec, openApiSpec } from './openApiSpec';
+import { execSync } from 'child_process';
+import { FUNCTIONS_JSON, OPEN_API_SPEC } from './functions.test';
+
+// If this test fails, it means the generated OpenAPI spec is no longer valid.
+test("generateValidSpec", async () => {
+    const apiSpec = generateOpenApiSpec(JSON.parse(FUNCTIONS_JSON));
+
+    const testFileName = "openApiSpec.test.yaml";
+    fs.writeFileSync(testFileName, apiSpec, 'utf-8');
+
+    let output = execSync(`npx redocly lint ${testFileName} --format='json'`);
+    console.log(output.toString());
+    expect(output.toString()).toContain(`"errors": 0`);
```

Comment:
is the output not JSON? could we avoid the stringify + matching?

also ideally we'd only print the output when the test fails (I like the philosophy of passing tests printing nothing, and only failing tests printing info)